setup.cfg

```
@@ -0,0 +1,47 @@
+[metadata]
+name = audbcards
+author = Hagen Wierstorf, Christian Geng, Baha Eddine Abrougui
```

My name is confusing I know 
```suggestion
author = Hagen Wierstorf, Christian Geng, BahaEddine Abrougui
```